bundles/org.openhab.binding.sonyprojector/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,195 @@
+
+	<channel-type id="contrastenhancer" advanced="true">
+		<item-type>String</item-type>
+		<label>Contrast Enhancer</label>
+		<description>Correct the level of bright and dark parts automatically</description>
+	</channel-type>
+
+	<channel-type id="filmmode" advanced="true">
+		<item-type>String</item-type>
+		<label>Film Mode</label>
+		<description>Select the playback mode for film sources</description>
+	</channel-type>
+
+	<channel-type id="gammacorrection" advanced="true">
+		<item-type>String</item-type>
+		<label>Gamma Correction</label>
+		<description>Adjust the response characteristics of the tone of the picture</description>
+	</channel-type>
+
+	<channel-type id="colorspace" advanced="true">
+		<item-type>String</item-type>
+		<label>Color Space</label>
+		<description>Convert the color space</description>
+	</channel-type>
+
+	<channel-type id="nr" advanced="true">
+		<item-type>String</item-type>
+		<label>Noise Reduction</label>
+		<description>Reduce the roughness or noise of the picture</description>
+	</channel-type>
+
+	<channel-type id="blocknr" advanced="true">
+		<item-type>String</item-type>
+		<label>Block Noise Reduction</label>
+		<description>Reduce digital noise that appears in mosaic-like pattern</description>
+	</channel-type>
+
+	<channel-type id="mosquitonr" advanced="true">
+		<item-type>String</item-type>
+		<label>Mosquito Noise Reduction</label>
+		<description>Reduce digital noise that appears near the outline of the picture</description>
+	</channel-type>
+
+	<channel-type id="mpegnr" advanced="true">
+		<item-type>String</item-type>
+		<label>MPEG Noise Reduction</label>
+		<description>Reduce block noise and mosquito noise, in particular in digital signals</description>
+	</channel-type>
+
+	<channel-type id="xvcolor" advanced="true">
+		<item-type>Switch</item-type>
+		<label>x.v.Color</label>
+		<description>Set this when playing back an x.v.Color video signal</description>
+	</channel-type>
+
+	<channel-type id="picturemuting" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Picture Muting</label>
+		<description>Mute or unmute the picture</description>
+	</channel-type>
+
+	<channel-type id="aspect" advanced="true">
+		<item-type>String</item-type>
+		<label>Aspect Ratio</label>
+		<description>Set the aspect ratio of the picture to be displayed</description>
+	</channel-type>
+
+	<channel-type id="overscan" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Overscan</label>
+		<description>Hide or not the edges of the picture</description>
+	</channel-type>
+
+	<channel-type id="pictureposition" advanced="true">
+		<item-type>String</item-type>
+		<label>Picture Position</label>
+		<description>Adjust the picture position</description>
+	</channel-type>
+
+	<channel-type id="lampusetime" advanced="true">
+		<item-type>Number</item-type>
+		<label>Lamp Use Time</label>
+		<description>Indicate how long the lamp has been turned on</description>
+		<state readOnly="true"></state>
```

Comment:
Shouldn't you provide a pattern that includes the unit?

Reply:
Pattern "%d h" added.